src/components/nav.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
+import * as navStyles from './nav.module.scss';
+import "@fontsource/righteous";
+import "@fontsource/roboto";
+
+const Nav = () => {
+  return (
+    <nav className={navStyles.nav}>
+      <div className={navStyles.logoContainer}>
+        <Link className={navStyles.logo} activeClassName={navStyles.activeLink}to="/">code2137</Link>
+      </div>
+      <ul className={navStyles.menu}>
+        <li className={navStyles.list}>
```

Comment:
SASS daje nam możliwość nestowania tzn możemy sobie zrobić w pliku scss .menu i potem już nie musimy tworzyć osobnej klasy dla list tylko możemy dodać zawartość w menu jako li. Podrzucam linka. To też tylko sugestia, ale wtedy mamy bardziej przejżysty kod i w scss i w komponentach. Oczywiście nie zawsze nestowanie jest dobre, czasami lepiej się odnaleźć w bardziej skomplikowanych rzeczach robiąc klasy dla każdego taga, ale nestowanie to generalnie fajne narzędzie. https://www.w3schools.com/sass/sass_nesting.php